src/channel/rpc_0_6.ts

```diff
@@ -448,6 +449,11 @@ export class RpcChannel {
           nonce: toHex(details.nonce),
         },
       });
+
+      logger.warn(
+        'You are using deprecated transaction version (V0,V1,V2)!, \n Update to the latest V3 transactions!',
```

Comment:
```suggestion
        'You are using a deprecated transaction version (V0,V1,V2)!\nUpdate to the latest V3 transactions!',
```
Might also make sense to move the message into a `const enum` so that it gets both defined in one place and inlined.